bin/fold

```diff
@@ -87,11 +90,11 @@ if ($Space_Break && $Width < $Tabstop) {
 }
 
 unshift(@ARGV, '-') unless @ARGV;
+my $err = 0;
 for (@ARGV) {
-    fold_file($_);
+    $err |= fold_file($_);
```

Comment:
I typically increment instead of toggling, but this is fine,. Haven't seen this in awhile. Fond memories. :)